lib/wallet/records.js

```diff
@@ -181,7 +168,7 @@ class BlockMeta extends bio.Struct {
    */
 
   getSize() {
-    return 42;
+    return 40;
```

Comment:
Digging in to this a bit to make sure that we don't break any serialized data already in users' DB. Since we use `StaticWriter` for `encode()` probably we were just allocating 2 more bytes than we need every time we write one of these, but those two extra bytes are just in memory and nothing will be changed how we read/write to disk... is that right?

Reply:
Actually, no. Serialization on this record is never called, all the usage and db entries related to this are partial. That's why it was there for so long. Otherwise any attempt to serialize BlockMeta would throw assertion error because of the offset/size not matching in the final `.write()` method for the serializer.

I am not 100% sure how I found it in the bcoin, most likely when working on bmultisig.

You can reproduce by setting getSize back to `42`, you will see `should encode/decode` of the `BlockMeta` test will fail with `Out of bounds write`.

Reply:
oh I see, we only ever write to DB `BlockMeta.toHash()` or height property, never serialize the whole object. OK!